lolcommits.gemspec

```diff
@@ -42,6 +42,7 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency('launchy', '~> 2.4.3')
   s.add_runtime_dependency('methadone', '~> 1.8.0')
   s.add_runtime_dependency('open4', '~> 1.3.4')
+  s.add_runtime_dependency('mercurial-ruby', '~> 0')
```

Comment:
I think its OK to have this dependency, like you said, it only requiring one other gem that we already have in our dep list.

Also, building our own internal version to parse mercurial is not something I think we should add to lolcommits.  Since we'd eventually have to keep on top of mercurial updates etc.